apps/web/components/team/TeamSettingsRightSidebar.tsx

```diff
@@ -118,7 +118,7 @@ export default function TeamSettingsRightSidebar(props: { team: TeamWithMembers;
         )}
       </div>
       {props.team?.id && props.role !== MembershipRole.MEMBER && (
-        <Link href={`/settings/teams/${props.team.id}/availability`}>
+        <Link href={`/settings/teams/${props.team.id}/availability`} passHref>
```

Comment:
Same here

Reply:
It wasn't a good idea to accept suggestions almost blindly, my bad.